### ISSUE TITLE
Anna Quindlen to do a good look when he saw was the only story you need to stop by a freckly cat head which popped from the wicked jaws of the bus: %30s The black fur with swirling grey on each side of the room, where he had a little backpack out of shells.

### ISSUE THREAD

**Issue by roadtojoy** [rossfuhrman/_why_the_lucky_markov#6656]
Toot: Anna Quindlen to do a good look when he saw was the only story you need to stop by a freckly cat head which popped from the wicked jaws of the bus: %30s The black fur with swirling grey on each side of the room, where he had a little backpack out of shells. 
 One comment = 1 upvote. Sometime after this gets 2 upvotes, it will be posted to the main account at https://mastodon.xyz/@_why_toots
